docs/upgrade.rst

```diff
@@ -0,0 +1,100 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Upgrade to Webpack
+==================
+
+In order to upgrade your module from AMD to Webpack, follow the steps below.
+
+Move files to the ``assets`` folder
+---------------------------------
+
+* Move files from ``static/js`` to the ``assets/js`` folder.
+* Move files from ``static/scss`` to the ``assets/scss`` folder.
+* Keep the rest of the static files in the ``static`` folder.
+
+Change the way of importing modules
+-----------------------------------
+Since Webpack doesn't use ``require.js``, you should change the way modules
+are imported in the JavaScript files. The example below shows how to
+import the modules:
+
+.. code-block:: javascript
+
+    import my-module from 'path/to/my/module'
+
+
+Create a WebpackBundle
+----------------------
+The `Flask-WebpackExt <https://flask-webpackext.readthedocs.io/en/latest/>`_
+package provides a class
+:class:`flask_webpackext.bundle.WebpackBundle` for declaring the needed
+assets and NPM dependencies of each one of your modules.
+
+.. code-block:: python
+
+    # webpack.py
+    from flask_webpackext import WebpackBundle
+    mybundle = WebpackBundle(
+        __name__,
+        './modules/module1/static',
+        entry={
+            'module1-app': './js/module1-app.js',
+        },
+        dependencies={
+            'jquery': '^3.2.1'
+        }
+    )
+
+The NPM dependencies defined in the bundles will be used to generate the
+``package.json`` file.
+
+This bundle will replace all of the previous bundles you have.
```

Comment:
maybe we can move this line before the `code-block` like:

> The `Flask-WebpackExt <https://flask-webpackext.readthedocs.io/en/latest/>`_
package provides a class
:class:`flask_webpackext.bundle.WebpackBundle` for declaring the needed
assets and NPM dependencies of each one of your modules. This class replaces the old bundles.

What do you think?